### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'add_job'

### ISSUE THREAD

**Issue by EDelsman** [thomasloven/hass-browser_mod#329]
My Home Assistant version: 2022.5.5

**What I am doing:**
Nothing in particular, below error in log happens every 5 seconds. Browser_mod is working as expected.

**What I expected to happen:**
Quiet in my log :-)

**What happened instead:**
Every 5 secs I get an error in the log that seems to be related to browser_mod. During (re)boot I do see two singular errors:
2022-05-22 13:58:41 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [547591161888] Received invalid command: browser_mod/connect
2022-05-22 13:58:42 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [547591161888] Received invalid command: browser_mod/update 

After that every 5 secs:
2022-05-22 14:19:12 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [547591161888] Error handling message: Unknown error (unknown_error)
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/connection.py", line 107, in async_handle
    handler(self.hass, self, schema(msg))
  File "/config/custom_components/browser_mod/connection.py", line 44, in handle_update
    devices[deviceID].update(msg.get("data", None))
  File "/config/custom_components/browser_mod/connection.py", line 94, in update
    self.sensor.data = data.get("browser")
  File "/config/custom_components/browser_mod/helpers.py", line 106, in data
    self.updated()
  File "/config/custom_components/browser_mod/sensor.py", line 27, in updated
    self.schedule_update_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 694, in schedule_update_ha_state
    self.hass.add_job(self.async_update_ha_state(force_refresh))
AttributeError: 'NoneType' object has no attribute 'add_job'

**Minimal** steps to reproduce:
None

```yaml
# The least amount of code or steps possible to reproduce my error

# Put your code/steps here
```

**Error messages from the browser console:**

```
// Select everything from the browser console and copy it

// Paste it here
```

---

**By replacing the space in the checkboxes ([ ]) with an X below, I indicate that I:**

- [X] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [X] Have made sure I am using the latest version of the plugin.

- [X] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [X] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by WallyDW** [comment]
Removing browser_mod from installed integrations stops the errors.

**Comment by alexreddy78** [comment]
@

> Removing browser_mod from installed integrations stops the errors.

What do you mean? Uninstall integration? But then, how you will use it?

**Comment by WallyDW** [comment]
It was not meant as a solution, but rather a confirmation that browser_mod seems to be the culprit.

**Comment by alexreddy78** [comment]
> culprit

Thanks, Cap

**Comment by thomasloven** [comment]
Closed because of Browser Mod 2.0 release.
Please see #354.
